app/views/submission_similarities/_student_history.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :head do %>
+  <%= javascript_include_tag "site_modal.js" %>
+  <%= javascript_include_tag "prettify/prettify.js" %>
+
+  <%= stylesheet_link_tag "prettify/prettify.css" %>
+<% end %>
+
+<%= render :template => 'layouts/pdf_report' %>
+
+<% @submission_similarity_ids.each do |id|%>
+  <% @submission_similarity = SubmissionSimilarity.find(id) %>
```

Comment:
I think there is a lack of separation of concern between view and controller here.